torchbox/torchbox.com
# Refactor footer logos to include mandatory link [BE]

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1399154333)

### Description of Changes Made

Based on the feedback received during the Sprint 3 demo today, this PR modifies the footer logos to include links, which are mandatory.

The link title is used as the logo alt text and anchor title attribute.

### How to Test

- In Wagtail Admin, go to [Settings > Navigation Settings](http://127.0.0.1:8000/admin/settings/navigation/navigationsettings/1/) and add a couple of footer logos.
- Attempting to save the settings without specifying a link should raise a validation error
- When you add the logos, they should be displayed on the template. When you put your mouse on the image, you should see the text for the link title (if a title wasn't specified, that is, for an internal link, then you should see the page title), and the image should be a clickable link.

### Screenshots

<details><summary>Wagtail Admin</summary>
  
![image](https://github.com/torchbox/torchbox.com/assets/7713776/94e438ad-9e64-4522-a13b-a02f451ca95f)

</details> 

<details>
  <summary>Validation Error</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/06ac4b4b-9505-4019-bcb1-80cd7555be56)

</details>

<details><summary>Template</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/4627a4d0-7835-4fd2-88bd-25eb9614e586)

</details> 


<details><summary>Pattern Library</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/bc005dc3-a96c-4195-a228-6dd0d05bac14)

</details> 

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on Ubuntu
  - Latest version of Firefox on Ubuntu

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
